### PR TITLE
Minor bug fixes ingress MS

### PR DESCRIPTION
**Pull request by ojahagirdar**
-Organize similar classes appropriately.
-OpenApiConfig changes.
-Manage DateUtil functions globally.
-Utilize SearchAPIPort.
-Remove the table's new name from ManageTable DTO.
-Remove ConfigSet-related classes & methods.
-Remove extra validations

### CODE REVIEW

File: src/test/java/com/searchservice/app/domain/service/ManageTableServiceTest.java [hunk 1]
```diff
@@ -585,17 +550,6 @@ void addSchemaAttributes() {
 
 	}
 
-	@Test
-	void checkDatesDifference() {
```
Comment (skdhitman) [inline src/test/java/com/searchservice/app/domain/service/ManageTableServiceTest.java:589]: Why this test case is removed?

**Review by skdhitman — CHANGES_REQUESTED** [review]
Please review all validations once.. esp. isTableExists if it'sgetting repeated, please remove

File: src/test/java/com/searchservice/app/domain/service/ManageTableServiceTest.java (hunk 1 above)
Comment (ojahagirdar) [reply src/test/java/com/searchservice/app/domain/service/ManageTableServiceTest.java:589]: it's already coverd

File: src/main/java/com/searchservice/app/domain/service/ManageTableService.java [hunk 2]
```diff
@@ -247,7 +240,9 @@ public Response deleteTable(String tableName) {
 
 		if (!isTableExists(tableName))
```
[the remark this answers is not in the document]
Comment (ojahagirdar) [reply src/main/java/com/searchservice/app/domain/service/ManageTableService.java:241]: validation  is removed

File: src/main/java/com/searchservice/app/domain/service/ManageTableService.java [hunk 3]
```diff
@@ -183,7 +170,9 @@ public TableSchemav2 getCurrentTableSchema(int tenantId, String tableName) {
 
 		if (!isTableExists(tableName + "_" + tenantId))
```
[the remark this answers is not in the document]
Comment (ojahagirdar) [reply src/main/java/com/searchservice/app/domain/service/ManageTableService.java:171]: yes

File: src/main/java/com/searchservice/app/domain/service/ManageTableService.java [hunk 4]
```diff
@@ -201,7 +190,8 @@ public TableSchemav2 getCurrentTableSchema(int tenantId, String tableName) {
 	public TableSchemav2 getTableSchemaIfPresent(String tableName) {
 
 		if (!isTableExists(tableName))
```
[the remark this answers is not in the document]
Comment (skdhitman) [reply src/main/java/com/searchservice/app/domain/service/ManageTableService.java:192]: ok 👍

File: src/main/java/com/searchservice/app/domain/service/ManageTableService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (skdhitman) [reply src/main/java/com/searchservice/app/domain/service/ManageTableService.java:241]: Alright 👍

File: src/test/java/com/searchservice/app/domain/service/ManageTableServiceTest.java (hunk 1 above)
Comment (skdhitman) [reply src/test/java/com/searchservice/app/domain/service/ManageTableServiceTest.java:589]: Oh, ok 👍

File: src/main/java/com/searchservice/app/domain/service/ManageTableService.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (ojahagirdar) [reply src/main/java/com/searchservice/app/domain/service/ManageTableService.java:171]: validation is in Use

**Pull request merged**